src/shared/rpc/methods/sign-psbt.ts

```diff
@@ -12,9 +12,21 @@ import {
   validateRpcParams,
 } from './validation.utils';
 
+// TODO: Replace when exported from btckit
+export enum SignatureHash {
+  ALL = 0x01,
+  NONE = 0x02,
+  SINGLE = 0x03,
+  ALL_ANYONECANPAY = 0x81,
+  NONE_ANYONECANPAY = 0x82,
+  SINGLE_ANYONECANPAY = 0x83,
+}
+
 const rpcSignPsbtParamsSchema = yup.object().shape({
   account: accountSchema,
-  allowedSighash: yup.array().of(yup.mixed().oneOf(Object.values(btc.SignatureHash))),
+  allowedSighash: yup
+    .array()
+    .of(yup.mixed().oneOf(Object.values(btc.SignatureHash || SignatureHash))),
```

Comment:
This isn't how `Object.values` works. This expression will always evaluate to `btc.SignatureHash` as it's defined.

To get the number values from the enum we need to filter as well, otherwise you allow string values.

ala

```ts
Object.values(enum).filter(Number.isInteger)
```

Reply:
O, right, that was dumb. Thanks. I should prob just add his two `0x00` and `0x80` to the enum here?

Reply:
Yeah that's works